core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -154,16 +163,20 @@ class Validator private (private val _name : String, val startStep : Step, val c
   }
 
   private class ValidatorFailGauge(private val timer : Timer,
-                                   private val failMeter : Meter) extends PercentGauge {
-
-    override def getNumerator = failMeter.oneMinuteRate
-    override def getDenominator = timer.oneMinuteRate
+                                   private val failMeter : Meter) extends RatioGauge {
+    override def getRatio: Ratio = Ratio.of(failMeter.oneMinuteRate, timer.oneMinuteRate)
   }
 
-  private val timer = metrics.timer(TIMER_NAME, name)
-  private val failMeter = metrics.meter(FAIL_METER_NAME, FAIL_METER_EVENT, name)
-  private val failGauge =  metricsRegistry.newGauge(getClass, FAIL_RATE_NAME, name,
-                                                   (new ValidatorFailGauge(timer, failMeter)))
+  private val timer = metrics.timer(name, TIMER_NAME)
+  private val failMeter = metrics.meter(MetricRegistry.name(name, FAIL_METER_NAME, FAIL_METER_EVENT))
+  Try {
+    metricRegistry.register(
+      MetricRegistry.name(getClass, name, FAIL_RATE_NAME),
+      new ValidatorFailGauge(timer, failMeter)
+    )
+  } recover {
+    case e: RuntimeException => logger.info("Problem adding new Fail gauge metric.", e)
```

Comment:
This was only happening during testing because the new registry would sometimes hang around in between tests.